tfx/components/trainer/fn_args_utils.py

```diff
@@ -86,6 +75,24 @@ def get_common_fn_args(input_dict: Dict[Text, List[types.Artifact]],
   json_format.Parse(exec_properties[constants.TRAIN_ARGS_KEY], train_args)
   json_format.Parse(exec_properties[constants.EVAL_ARGS_KEY], eval_args)
 
+  # Default behavior (when train_args.splits and eval_args.splits are not set)
+  # is train on ‘train’ splits and evaluate on ‘eval’ splits.
+  if not train_args.splits:
+    train_args.splits = 'train'
```

Comment:
is splits a list of str?

keep existing test cases (covers default split code path), and create a new test covers custom split path